content/en/Product Updates/release-notes-april-2024.md

```diff
@@ -3,11 +3,10 @@ title: "Release Notes: April 2024"
 linkTitle: "April 2024"
 weight: 48
 description: >-
-     Explore What's New from Cobalt This Month
 ---
 
 {{% pageinfo %}} 
-Stay Up-to-Date with the Latest Enhancements on the Cobalt Platform. 
+     Explore What's New from Cobalt This Month
```

Comment:
I see that the blue info box is updated only for April, August, and September 2024. I wasn’t sure which style was intended or you intended to just remove the subtitle and descriptions.
<img width="559" alt="Screenshot 2024-11-01 at 11 48 33" src="https://github.com/user-attachments/assets/e759d7c1-8861-4ccf-8035-d1cededc1dcb">

<img width="542" alt="Screenshot 2024-11-01 at 11 48 37" src="https://github.com/user-attachments/assets/0138418c-a676-4786-9223-7121ae96fea8">

Reply:
@mtsuji-cobalt The solid blue box is definitely what I am going for, I think when the text was indented it started to create that inner white box thing. I just went through and updated all the formats so this should be resolved.